packages/ibm-products/src/components/EmptyStates/EmptyStates.stories.jsx

```diff
@@ -55,14 +96,7 @@ Default.args = {
 export const WithTooltipInSubtitle = Template.bind({});
 WithTooltipInSubtitle.args = {
   ...emptyStateCommonProps,
-  subtitle: (
-    <>
-      Click <span>here</span> to upload your data
-      <Tooltip label="Facts and statistics collected together for reference or analysis">
-        <Information size="16" />
-      </Tooltip>
-    </>
-  ),
+  subtitle: 1,
```

Comment:
i noticed that when you check the controls in storybook that `subtitle` just says `1` so we end up obfuscating these inputs. lets try separating the getter functions from the template and see if we can populate these controls